test/games/dread/exporter/test_dread_patch_data_factory.py

```diff
@@ -336,7 +337,7 @@ def _test_preset(rdvgame_file: Path, expected_results_file: Path):
         ("dread_prime1_multiworld.rdvgame", "dread_prime1_multiworld_expected.json"),  # dread-prime1 multi
     ]
 )
-def test_create_patch_data(test_files_dir, rdvgame_filename, expected_results_filename):
+def test_create_rdvgame_files(test_files_dir, rdvgame_filename, expected_results_filename):
```

Comment:
the old name is more accurate, since we're using rdvgame to create patch data

Reply:
Yeah, I can rename it again but we have two `test_create_patch_data` methods in this file. The very first and the very last that's why I renamed one.

Reply:
These two tests can very much be combined into one

Reply:
It's kinda ot here. See #4552